Duvet/Configuration/SheetConfiguration.swift

```diff
@@ -25,6 +25,11 @@ public struct SheetConfiguration: Equatable {
     /// The number of points that the sheet should sit below the top safe area.
     public var topInset: CGFloat
 
+    /// A flag indicating if this sheet should use Duvet's default keyboard avoidance or not. If `true`,
+    /// Duvet will handle keyboard avoidance, otherwise it will do nothing when the keyboard is presented
+    /// or dismissed.
+    public var usesDefaultKeyboardAvoidance: Bool
```

Comment:
Minor nitpick, but `usesDefaultKeyboardAvoidance` makes it sounds like to me that there are other types of keyboard avoidance that could be used. Maybe just `enableKeyboardAvoidance`?

Reply:
I like this! I renamed `usesDefaultKeyboardAvoidance` to `isKeyboardAvoidanceEnabled`.